src/lollipopChart.js

```diff
@@ -385,6 +411,12 @@ var LollipopChart = function (selection) {
     if(chart.isBadDatum(d)) return svgHeight + lollipopRadius;
     return svgHeight - chart.generateLollipopHeight(d);
   }
+  
+  function generateLollipopLabelAxisX(d, i) {
+    var barX = generateBarX(d, i);
+    
+    return barX - barGap * (1/16);
```

Comment:
Yeah, the 1/16 value should work no matter what data is given to the chart, but it might be better suited to a parameter. The distance from each bar scales with the gap between the bars. Like the rectangles, the label axis doesn't scale to the lollipop size, so if it gets too narrow the axis will overlap the lollipops:
<img width="496" alt="screen shot 2017-03-17 at 11 06 00 am" src="https://cloud.githubusercontent.com/assets/3695899/24049934/cde8ed8c-0b03-11e7-909f-6f0d68844c9b.png">

Number of bars reduced:
<img width="475" alt="screen shot 2017-03-17 at 11 08 12 am" src="https://cloud.githubusercontent.com/assets/3695899/24049960/df824dae-0b03-11e7-9496-b117fbc1968b.png">

barGap increased from 35 (default) to 95:
<img width="488" alt="screen shot 2017-03-17 at 11 17 01 am" src="https://cloud.githubusercontent.com/assets/3695899/24049990/f38a61ba-0b03-11e7-8832-dfa8e2414c3f.png">